charts/em-ccd-orchestrator/values.aat.template.yaml

```diff
@@ -2,3 +2,4 @@ ccd-orchestrator:
   # Don't modify below here
   image: ${IMAGE_NAME}
   ingressHost: ${SERVICE_FQDN}
+  aadIdentityName:
```

Comment:
Shouldn't we delete this so that the default value comes through from values.yaml? I believe it only should be removed in the preview env

Reply:
Agree, I'll modify this.